truss/templates/base.Dockerfile.jinja

```diff
@@ -45,11 +49,11 @@ RUN apt-get update && apt-get install --yes --no-install-recommends $(cat {{ sys
 {% block install_requirements %}
     {%- if should_install_user_requirements_file %}
 COPY ./{{ user_supplied_requirements_filename }} {{ user_supplied_requirements_filename }}
-RUN pip install -r {{ user_supplied_requirements_filename }} --no-cache-dir && rm -rf /root/.cache/pip
+RUN {{ sys_pip_install_command }} -r {{ user_supplied_requirements_filename }} --no-cache-dir && rm -rf /root/.cache/pip
```

Comment:
From brief research, it seems like `--no-cache-dir` replaces the need for `rm -rf /root/.cache/pip` entirely, so I removed those as well